src/screens/AddNewItemScreen/index.tsx

```diff
@@ -64,10 +81,16 @@ export const AddNewItemScreen: FC = () => {
     });
   }, [onPressBack, navigation]);
 
-  const onPress = () => {
+  const onPressAdd = () => {
     addItem(text);
     navigation.goBack();
   };
+  const onPressEdit = () => {
+    dispatch(editItemSagaActoin({ id: editItem.id, text }));
```

Comment:
where is loader? what if this request fails, user should go back to previous screen?